docs/developers/developing_iree/testing_guide.md

```diff
@@ -3,6 +3,96 @@
 Like the IREE project in general, IREE tests are divided into a few different
 components and use different tooling depending on the needs of that component.
 
+| Test type       | Test                                            | Build system | Supported platforms              |
+|:--------------  | :-----------------                              | -----------  | -------------                    |
+| Compiler tests  | iree_lit_test                                   | Bazel/CMake  | Host                             |
+|                 | iree_lit_test_suite                             | Bazel/CMake  | Host                             |
```

Comment:
You can omit the `*_test_suite` "test types" where there is a corresponding `*_test` from this table IMO - a suite is just a grouping of tests

Reply:
Done. I leave `iree_hal_cts_test_suite` in the table, as it does not have a single test. I also put some note of `*_test_suite` after the table.